layers/+lang/java/packages.el

```diff
@@ -462,7 +462,12 @@
         "rem" 'lsp-java-extract-method
         "cc"  'lsp-java-build-project
         "an"  'lsp-java-actionable-notifications
-        "="   'lsp-format-buffer)
+        "="   'lsp-format-buffer
+
+        ;; dap-mode
+        "xj"   'dap-java-debug
+        "tt"   'dap-java-debug-test-method
```

Comment:
This is used in other modes as "Run only the current test at cursor position" and I would like to keep it that way here as well. Can you think of another key binding?

Reply:
What do you think about adding no-debug prefix for the command so `SPCu` `SPCmtt` would start without debugging?

Reply:
I would rather this keybinding live under the `debug` prefix as well, maybe `, d t t` in case you want to put more keybindings next to it.